src/components/SelectionList/BaseSelectionList.js

```diff
@@ -394,7 +394,7 @@ function BaseSelectionList({
         >
             <SafeAreaConsumer>
                 {({safeAreaPaddingBottomStyle}) => (
-                    <View style={[styles.flex1, !isKeyboardShown && safeAreaPaddingBottomStyle]}>
+                    <View style={[styles.flex1, !isKeyboardShown && safeAreaPaddingBottomStyle, {opacity: isInitialRender ? 0 : 1}]}>
```

Comment:
Maybe it's better to extract `{opacity: isInitialRender ? 0 : 1}` to a variable instead of re-creating the object here every time?